packages/amplify-provider-awscloudformation/src/configuration-manager.js

```diff
@@ -509,18 +541,81 @@ function loadConfigFromPath(context, profilePath) {
   throw new Error(`Invalid config ${profilePath}`);
 }
 
-async function loadConfigurationForEnv(context, env) {
+async function getAdminCredentials(idToken, identityId, region) {
+  const cognitoIdentity = new aws.CognitoIdentity({ region });
+  const login = idToken.payload.iss.replace('https://', '');
+  return cognitoIdentity
+    .getCredentialsForIdentity({
+      IdentityId: identityId,
+      Logins: {
+        [login]: idToken.jwtToken,
+      },
+    })
+    .promise();
+}
+
+async function loadConfigurationForEnv(context, env, appId) {
   const projectConfigInfo = getConfigForEnv(context, env);
-  if (projectConfigInfo.configLevel === 'project') {
+  const { print } = context;
+  let awsConfig;
+  if (projectConfigInfo.configLevel === 'amplifyAdmin' || appId) {
+    projectConfigInfo.configLevel = 'amplifyAdmin';
+    appId = appId || stateManager.getMeta().providers.awscloudformation.AmplifyAppId;
```

Comment:
Can we use `_.get` to make sure we are not missing any checks in the object chain?

Reply:
Changed to _.get.